streamz/dataframe/tests/test_dataframes.py

```diff
@@ -755,12 +755,11 @@ def f(x):
     assert len(L) == 5
 
     first = df.iloc[:diff]
-    lost = first[first.index.min() + value:]
-    first = first.iloc[len(lost):]
+    first = first[first.index.max() - value + pd.Timedelta('1ns'):]
 
     assert_eq(L[0], f(first))
 
-    last = df.loc[index.max() - value + pd.Timedelta('1s'):]
+    last = df.loc[index.max() - value + pd.Timedelta('1ns'):]
```

Comment:
Should include your specific test case in the suite?

Reply:
Done